atom/browser/api/atom_api_system_preferences_mac.mm

```diff
@@ -106,10 +106,18 @@ AVMediaType ParseMediaType(const std::string& media_type) {
 
 }  // namespace
 
-void SystemPreferences::PostNotification(
-    const std::string& name,
-    const base::DictionaryValue& user_info) {
-  DoPostNotification(name, user_info, kNSDistributedNotificationCenter);
```

Comment:
can we delete the `NotificationCenterKind` enum now?

Reply:
yes! will do